packages/terra-clinical-detail-view/src/DetailView.scss

```diff
@@ -9,24 +9,61 @@
 
   .title {
     line-height: 1.2;
-    padding-bottom: 0.7142857143rem;
+    margin-bottom: 0.7142857143rem;
+
+    &:first-child {
+      margin-bottom: 1.4285714286rem;
```

Comment:
Curious, what's the reasoning behind flexing `margin-bottom` if title contains a child, versus keeping it the same value either way?

Reply:
I guess, Neil suggested this way, because, when any of the atomic props are missing (say accessory), without the first-child, it is going to create a lot of space (if margin-bottom is coded to take 1.4285714286rem by default). In such scenarios, it should take 0.714... rem to avoid more gaps. Hope this answered the question.

Reply:
Correct - the design spacing flexes based on which elements are present or not present to have a more legible layout. We took a bit of time working out the most elegant mix and match of styling and this seems to be the simplest. When we add theme able variables, the code will be tested again, but I feel pretty good about the current approach.